lexos/static/js/scripts_viz.js

```diff
@@ -174,7 +174,7 @@ $(window).on('load', function () {
       canvas = document.getElementById('svg-canvas')
 
     // Draw svg on canvas
-    canvg(canvas, content)
+    //canvg(canvas, content)
```

Comment:
please explain why this code is commented out.

Reply:
I see your comment saying that it is unnecessary. I am not familiar with this code. Maybe Scott knows why?

Reply:
For now we can ignore this. It was just something I noticed that is unrelated to the jQuery update. This can be something that someone investigates more when this page is refactored.

Reply:
It looks to me like this is a remnant of a former method of rendering a .png file using `canvg.js`. I would revise the code to this:

```javascript
  // Save to PNG
  $('#save').on('click', function () {
    /* Code may be obsolete/deprecated -- check to see if a PNG is generated without canvg.js */
   /*
    var $container = $('#viz'),
    // Canvg requires trimmed content
    content = $container.html().trim(), // Note trailing comma here. Broken code?
    // Draw svg on canvas
    canvg(canvas, content)
    */

    canvas = document.getElementById('svg-canvas')

    // Change img from SVG representation
    var theImage = canvas.toDataURL('image/png')
    $('#svg-img').attr('src', theImage)

    // Open a new window with the image
    var w = window.open()
    var img = $('#svg-img').clone().css('display', 'block')
    var html = $('<div/>')
    html.append("<h3>Use your browser's Save as function to save the image</h3>")
    html.append(img)
    $(w.document.body).html(html)
    // End Save
  })
})
```
And perhaps open up an issue to check the PNG generating function.